doc/src/SUMMARY.md

```diff
@@ -1,19 +1,13 @@
 [Introduction](README.md)
 
+[Contributing guide](contributing.md)
```

Comment:
It might be better to place the contibuting information at the bottom, as previously done, since most people will never read this. The more commonly used information should be at the top.